tools/lib/tests/test_caching.py

```diff
@@ -34,6 +35,28 @@ def do_HEAD(self):
 
 class TestFileDownload(unittest.TestCase):
 
+  def test_defaults(self):
+    # TODO: parameterize the defaults so we don't rely on hard-coded values in xx
+
+    self.assertEqual(URLFile.pool_manager().pools._maxsize, 10) # PoolManager num_pools param
```

Comment:
can you add a comment that makes it clear that these defaults are for the pipeline? or maybe put it in the test name?